src/client/lib/utils.js

```diff
@@ -54,7 +54,7 @@ export var lookup = (data, key) => {
 };
 
 
-var run = (g, cb) => {
+let run = (g, cb) => {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).